### ISSUE TITLE
The performance of OCR detection model become slower when enable mkldnn

### ISSUE THREAD

**Issue by juncaipeng** [PaddlePaddle/Paddle#28554]
Thank you for contributing to PaddlePaddle.
Before submitting the issue, you could search issue in the github in case that th
If there is no solution,please make sure that this is an inference issue including the following details :
**System information**
-PaddlePaddle version: develop (commit id: 72c78e4dbbdf74fe9995d9b0757ca6e5832c79d7) or 1.8.5
-CPU: 6148 with MKLDNN
-GPU: None
-OS Platform:Ubuntu 16.04.5
-Python version:3.7
Note: You can get most of the information by running [summary_env.py](https://github.com/PaddlePaddle/Paddle/blob/develop/tools/summary_env.py). 
**To Reproduce**
Steps to reproduce the behavior
**Describe your current behavior**
**Code to reproduce the issue**
1. Download the code and model, url: https://pan.baidu.com/s/1X5jdXj6Vq4dL1Vkk_yGplw  password: kv1y

2. Refer to readme.md and run the program

![image](https://user-images.githubusercontent.com/52520497/98806947-e7382000-2454-11eb-9b9e-9f932e54849e.png)
![image](https://user-images.githubusercontent.com/52520497/98807354-7d6c4600-2455-11eb-899c-0b2dca7c53bb.png)

3. Modify the config to disable mkldnn and run the program

![image](https://user-images.githubusercontent.com/52520497/98806920-df787b80-2454-11eb-9a4c-62af16c638ae.png)

![image](https://user-images.githubusercontent.com/52520497/98807322-6f1e2a00-2455-11eb-9283-460216a8d0ac.png)

**Other info / logs**

**Comment by lidanqing-vv** [comment]
model is downloaded at `shanghai-6271-ip:/data/issue_28554` (for intel colleague)

**Comment by juncaipeng** [comment]
The OCR recognition model also has the same problem. 
The demo can be downloaded from #28578 , and only consider the fp32 model.

**Comment by lidanqing-vv** [comment]
Fix PR: https://github.com/PaddlePaddle/Paddle/pull/29103. I am still verifying accurcay
Native
```
------------------------->     Profiling Report     <-------------------------

Place: CPU
Time unit: ms
Sorted by total time in descending order in the same thread

-------------------------     Overhead Summary      -------------------------

Total time: 7622.93
  Computation time       Total: 7594.25     Ratio: 99.6238%
  Framework overhead     Total: 28.6751     Ratio: 0.376171%

-------------------------     GpuMemCpy Summary     -------------------------

GpuMemcpy                Calls: 0           Total: 0           Ratio: 0%

-------------------------       Event Summary       -------------------------

Event                            Calls       Total       Min.        Max.        Ave.        Ratio.
thread0::conv2d                  820         3455.21     0.125731    94.5211     4.21367     0.453265
thread0::depthwise_conv2d        300         2242.45     0.792492    19.227      7.47482     0.294172
thread0::elementwise_add         960         616.811     0.029866    10.5128     0.642512    0.0809153
thread0::nearest_interp          120         342.694     0.329599    14.4631     2.85579     0.0449557
thread0::relu                    280         294.54      0.205798    8.82413     1.05193     0.0386387
thread0::conv2d_transpose        40          249.65      2.09722     14.8893     6.24124     0.0327499
thread0::hard_swish              400         132.984     0.093146    4.37311     0.33246     0.0174453
thread0::batch_norm              300         119.893     0.116536    2.12391     0.399644    0.015728
thread0::concat                  20          85.6974     3.60962     9.00092     4.28487     0.0112421
thread0::load_combine            1           51.1563     51.1563     51.1563     51.1563     0.00671085
thread0::sigmoid                 20          22.4155     1.01457     1.49151     1.12077     0.00294054
thread0::scale                   20          9.42836     0.380054    0.801661    0.471418    0.00123684
```

Enable mkldnn
```
------------------------->     Profiling Report     <-------------------------

Place: CPU
Time unit: ms
Sorted by total time in descending order in the same thread

-------------------------     Overhead Summary      -------------------------

Total time: 4090.38
  Computation time       Total: 3700.17     Ratio: 90.4602%
  Framework overhead     Total: 390.215     Ratio: 9.53981%

-------------------------     GpuMemCpy Summary     -------------------------

GpuMemcpy                Calls: 0           Total: 0           Ratio: 0%

-------------------------       Event Summary       -------------------------

Event                            Calls       Total       Min.        Max.        Ave.        Ratio.
thread0::conv2d                  1120        1821.36     0.178811    25.2532     1.62622     0.445279
thread0::conv2d_transpose        40          640.786     8.43141     33.99       16.0196     0.156657
thread0::sigmoid                 20          428.068     19.968      33.1246     21.4034     0.104652
thread0::nearest_interp          120         344.835     0.439659    12.062      2.87362     0.0843037
thread0::hard_swish              400         276.7       0.1856      11.4515     0.691749    0.0676464
thread0::scale                   20          207.606     10.1698     11.3474     10.3803     0.0507546
thread0::relu                    20          129.93      5.82949     11.9667     6.49652     0.0317648
thread0::concat                  20          105.236     4.51194     10.9182     5.26179     0.0257276
thread0::elementwise_add         60          88.6055     0.190968    7.04085     1.47676     0.0216619
thread0::load_combine            1           47.2565     47.2565     47.2565     47.2565     0.0115531
```

**Comment by juncaipeng** [comment]
Interpolate mkldnn op has been merged to paddle 2.0, the performance of ocr detection model is normal.
